fxmanifest.lua

```diff
@@ -41,13 +41,8 @@ server_scripts {
 
 files {
     'modules/*.lua',
-    'shared/gangs.lua',
-    'shared/items.lua',
-    'shared/jobs.lua',
-    'shared/locations.lua',
+    'data/*.lua',
```

Comment:
also this is no longer needed currently

Reply:
good catch. i'm not opposed to either solution. just kind of needs to be in it's own place away from actual "code" that needs to execute